### PR TITLE
Add example for fiedler_vector

### PR DESCRIPTION
**Pull request by paulitapb**
I added an example for `fiedler_vector` in `algebraicconnectivity.py`. In the example, I use the Fiedler vector to partition a connected graph.

### CODE REVIEW

**Comment by jarrodmillman** [comment]
I would choose a different graph. Normally the idea of bipartitioning a graph is that there are two components that have more internal edges than external edges. A cycle has no natural partitioning.

File: networkx/linalg/algebraicconnectivity.py [hunk 1]
```diff
@@ -460,6 +460,17 @@ def fiedler_vector(
...
+    Examples
+    --------
+    Given a connected graph the signs of the values in the Fiedler vector can be
+    used to partition the graph into two components.
+
+    >>> G = nx.cycle_graph(4)
+    >>> print(nx.fiedler_vector(G, normalized=True, seed = 1))
+    [-0.69141345 -0.1481467   0.69141344  0.14814671]
+
+    The connected components are {0,1} and {2,3}.
```
Comment (jarrodmillman) [inline networkx/linalg/algebraicconnectivity.py:473]:
```suggestion
    >>> G = nx.barbell_graph(5, 0)
    >>> nx.fiedler_vector(G, normalized=True, seed=1)
    array([-0.32864129, -0.32864129, -0.32864129, -0.32864129, -0.26072899,
            0.26072899,  0.32864129,  0.32864129,  0.32864129,  0.32864129])
    >>> nx.fiedler_vector(G, normalized=True, seed=1)
    array([-0.32864129, -0.32864129, -0.32864129, -0.32864129, -0.26072899,
            0.26072899,  0.32864129,  0.32864129,  0.32864129,  0.32864129])

    The connected components are the two 5-node cliques of the barbell graph.
```

**Comment by jarrodmillman** [comment]
I suggested an example.

File: networkx/linalg/algebraicconnectivity.py (hunk 1 above)
Comment (paulitapb) [reply networkx/linalg/algebraicconnectivity.py:473]: Yes, that's true. I also thought of the karate club graph but wanted a small example.  This is perfect!

**Review by MridulS — APPROVED** [review]
thanks @paulitapb!

**Review by rossbar — APPROVED** [review]
LGTM, thanks @paulitapb & @jarrodmillman !

**Pull request merged**